ui/cypress/e2e/onboarding.test.ts

```diff
@@ -0,0 +1,423 @@
+interface TestUser {
+  username: string
+  password: string
+  org: string
+  bucket: string
+}
+
+describe('Onboarding', () => {
+  let user: TestUser
+
+  beforeEach(() => {
+    cy.flush()
+
+    cy.fixture('user').then(u => {
+      user = u
+    })
+
+    cy.visit('/')
+  })
+
+  it('Can Onboard to Quick Start', () => {
+    cy.server()
+
+    //Will want to capture response from this
+    cy.route('POST', 'api/v2/setup').as('orgSetup')
+
+    //Check splash page
+    cy.location('pathname').should('include', 'onboarding/0')
+    cy.getByTestID('init-step--head-main').contains('Welcome to InfluxDB 2.0')
+    cy.getByTestID('credits').contains('Powered by')
+    cy.getByTestID('credits').contains('InfluxData')
+
+    //Continue
+    cy.getByTestID('button').click()
+
+    cy.location('pathname').should('include', 'onboarding/1')
+
+    //Check navigation bar
+    cy.getByTestID('nav-step--welcome').click()
+
+    //Check splash page
+    cy.getByTestID('init-step--head-main').contains('Welcome to InfluxDB 2.0')
+    cy.getByTestID('credits').contains('Powered by')
+    cy.getByTestID('credits').contains('InfluxData')
+
+    //Continue
+    cy.getByTestID('button').click()
+
+    //Check onboarding page - nav bar
+    cy.getByTestID('nav-step--welcome').contains('Welcome')
+    cy.getByTestID('nav-step--welcome')
+      .parent()
+      .children('span')
+      .should($span => {
+        expect($span).to.have.class('checkmark')
+      })
+
+    cy.getByTestID('nav-step--setup')
+      .contains('Initial User Setup')
+      .should('be.visible')
+    cy.getByTestID('nav-step--setup').should('have.class', 'current')
+    cy.getByTestID('nav-step--setup')
+      .parent()
+      .children('span')
+      .children('span')
+      .should($span => {
+        expect($span)
+          .to.have.class('icon')
+          .and.to.have.class('circle-thick')
+      })
+
+    cy.getByTestID('nav-step--complete')
+      .parent()
+      .should($el => {
+        expect($el).to.have.class('unclickable')
+      })
+
+    //Check onboarding page headers and controls
+    cy.getByTestID('admin-step--head-main').contains('Setup Initial User')
+
+    cy.getByTestID('button')
+      .contains('Continue')
+      .should('be.disabled')
+
+    //Check tooltips
+    cy.getByTestID('form--label')
+      .eq(3)
+      .children('div.question-mark-tooltip')
+      .trigger('mouseenter')
+      .children('div[data-id=tooltip]')
+      .contains('An organization is')
+      .should($tt => {
+        expect($tt).to.have.class('show')
+      })
+
+    cy.getByTestID('form--label')
+      .eq(4)
+      .children('div.question-mark-tooltip')
+      .trigger('mouseenter')
+      .children('div[data-id=tooltip]')
+      .contains('A bucket is')
+      .should($tt => {
+        expect($tt).to.have.class('show')
+      })
+
+    //Input fields
+    cy.getByTestID('input-field')
+      .eq(0)
```

Comment:
It would be more resilient to use the testID prop to inputs rather than the default testID here.   The position of the inputs is highly subject to change.

Reply:
I've added more data-testID attribs and used them to get around using indexed list items